texar/custom/__init__.py

```diff
@@ -4,3 +4,4 @@
 
 from texar.custom.activation import *
 from texar.custom.initializers import *
+from texar.custom.distributions import *
```

Comment:
I believe you forgot to commit `distributions.py`.

Reply:
Indeed! Thanks

Reply:
Corrected.